### PR TITLE
Update Maintenance slides

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

**Comment by nsoranzo** [comment]
Should we remove the duplicated material from https://training.galaxyproject.org/training-material/topics/admin/tutorials/object-store/slides.html ?

Not necessarily part of this PR, it can be added to https://github.com/galaxyproject/training-material/issues/1809

**Comment by nsoranzo** [comment]
Should we remove the duplicated material from https://training.galaxyproject.org/training-material/topics/admin/tutorials/object-store/slides.html ?

Not necessarily part of this PR, it can be added to https://github.com/galaxyproject/training-material/issues/1809

**Comment by nsoranzo** [comment]
Uh, and also from topics/admin/tutorials/stuff/slides.html

File: topics/admin/tutorials/maintenance/slides.html [hunk 1]
```diff
@@ -2,98 +2,163 @@
...
   - hexylena
 ---
+class: left
 
 # Runaway Storage
 
 Tips:
 - Set quotas
-- `tmpwatch` your job working directory
+- `tmpwatch` (`tmpreaper` on Debianish) your job working directory
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
- `tmpwatch` (`tmpreaper` on Debian-based distros) your job working directory
```

**Comment by natefoo** [comment]
> Should we remove the duplicated material

Oof, didn't realize this.

File: topics/admin/tutorials/maintenance/slides.html [hunk 2]
```diff
@@ -2,98 +2,163 @@
...
 ---
+class: left
 
 # Runaway Storage
 
 Tips:
 - Set quotas
-- `tmpwatch` your job working directory
+- `tmpwatch` (`tmpreaper` on Debian-based distros) your job working directory
   - `cleanup_job` in `galaxy.yml` (defaults to `always` though)
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]: I normally change it to `onsuccess` to be able to debug user issue reports.

File: topics/admin/tutorials/maintenance/slides.html [hunk 3]
```diff
@@ -2,98 +2,163 @@
...
+- `tmpwatch` your `new_file_path`
+  - `/usr/bin/tmpwatch -v --mtime --dirmtime 7d /srv/galaxy/var/tmp`
 - Set up dataset cleanup
 
 ---
 # Dataset Cleanup
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
## Dataset Cleanup
```

File: topics/admin/tutorials/maintenance/slides.html [hunk 4]
```diff
@@ -2,98 +2,163 @@
...
+| `scripts/cleanup_datasets/pgcleanup.py` | PostgreSQL-optimized fast cleanup script |
+| `scripts/cleanup_datasets/cleanup_datasets.py` | General cleanup script |
+| `gxadmin cleanup <days>` | calls pgcleanup |
 
 ---
-# Dataset Cleanup Lifecycle
+class: reduce70
 
-Mark deleted all "anonymous" histories not used within the last `$days` days:
+# pgcleanup invocation
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
## pgcleanup invocation
```

File: topics/admin/tutorials/maintenance/slides.html (hunk 2 above)
Comment (natefoo) [reply topics/admin/tutorials/maintenance/slides.html]: Same, which is why you might need `tmpwatch` (you don't if you leave `cleanup_job` at the default).

File: topics/admin/tutorials/maintenance/slides.html [hunk 5]
```diff
@@ -2,98 +2,163 @@
...
 
-Remove all history exports older than `$days` days:
+---
+class: reduce70
 
-```console
-python ./scripts/cleanup_datasets/pgcleanup.py \
-    -o $days -s delete_exported_histories
-```
+# pgcleanup actions
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
## pgcleanup actions
```

File: topics/admin/tutorials/maintenance/slides.html [hunk 6]
```diff
@@ -2,98 +2,163 @@
...
+| `purge_deleted_histories` | <ul><li>Mark purged all Histories marked deleted that are older than the specified number of days.</li><li>Mark purged all HistoryDatasetAssociations in Histories marked purged in this step (if not already purged).</li></ul> |
+| `purge_deleted_hdas` | <ul><li>Mark purged all HistoryDatasetAssociations currently marked deleted that are older than the specified number of days.</li><li>Mark deleted all MetadataFiles whose hda_id is purged in this step.</li><li>Mark deleted all ImplicitlyConvertedDatasetAssociations whose hda_parent_id is purged in this step.</li><li>Mark purged all HistoryDatasetAssociations for which an ImplicitlyConvertedDatasetAssociation with matching hda_id is deleted in this step.</li></ul> |
 
 ---
+class: reduce70
 
-# Dataset Cleanup Lifecycle
+# More pgcleanup actions
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
## More pgcleanup actions
```

File: topics/admin/tutorials/maintenance/slides.html [hunk 7]
```diff
@@ -2,98 +2,163 @@
...
+class: left, reduce90
 
-Mark datasets with all purged HDAs last updated `$days` or more days ago deleted:
+# Backups
 
-```console
-python ./scripts/cleanup_datasets/pgcleanup.py \
-    -o $days -s delete_datasets
-```
+Restorable from Ansible Playbook:
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]: Move this to the "What not to back up" section below (maybe still as a separate table, but a dedicated slide for "What not to back up").

File: topics/admin/tutorials/maintenance/slides.html [hunk 8]
```diff
@@ -2,98 +2,163 @@
...
+| ---- | ---- |
+| Anything in *managed/mutable data dir* not mentioned above | `/srv/galaxy/var` |
+| Job working directories | `/srv/galaxy/jobs` |
+
+[postgresql-pitr]: https://www.postgresql.org/docs/current/continuous-archiving.html
+[ansible-postgresql]: https://github.com/galaxyproject/ansible-postgresql
+
+---
+
+# Restoring from backups
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
## Restoring from backups
```

File: topics/admin/tutorials/maintenance/slides.html [hunk 9]
```diff
@@ -2,98 +2,163 @@
...
 
-```console
-python ./scripts/cleanup_datasets/pgcleanup.py \
-    -o $days -s purge_deleted_hdas
-```
+---
+class: left, reduce90
 
-Mark datasets with all purged HDAs last updated `$days` or more days ago deleted:
+# Backups
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]:
```suggestion
# Backups

--

## What to backup
```

**Pull request merged**